pkg/versionservice/versionservice.go

```diff
@@ -0,0 +1,138 @@
+package versionservice
+
+import (
+	"context"
+	"fmt"
+	apiv1alpha1 "github.com/percona/percona-server-mysql-operator/api/v1alpha1"
+	"github.com/percona/percona-server-mysql-operator/pkg/platform"
+	vsclient "github.com/percona/percona-server-mysql-operator/pkg/versionservice/client"
+	"github.com/percona/percona-server-mysql-operator/pkg/versionservice/client/models"
+	"github.com/percona/percona-server-mysql-operator/pkg/versionservice/client/version_service"
+	"github.com/pkg/errors"
+	"net/http"
+	"net/url"
+	"time"
+
+	_ "github.com/Percona-Lab/percona-version-service/api"
+)
+
+const (
+	productName     = "ps-operator"
+	defaultEndpoint = "https://check.percona.com"
+)
+
+func GetExactVersion(ctx context.Context, cr *apiv1alpha1.PerconaServerMySQL, serverVersion *platform.ServerVersion) (DepVersion, error) {
```

Comment:
Can we simply name it as `GetVersion` because I guess that we can only get an exact version (or error)?

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/151/commits/3336703af098a0467efa581d3f03f9a2de4a4c82